client/app.jsx

```diff
@@ -1,14 +1,15 @@
 import React from 'react';
 import UserCardList from './pages/user-card-list';
-import Header from './components/header';
 import Profile from './pages/profile';
 import NoContent from './pages/no-content';
 import parseRoute from './lib/parse-route';
+import Chatroom from './pages/chat-room';
 
 export default class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
+      currentUser: 5,
```

Comment:
You can remove this and make all assumptions about the logged in user on the server, since we will be using tokens to determine that later on.